packages/jest-cli/src/EmptySuiteError.js

```diff
@@ -0,0 +1,9 @@
+class EmptySuiteError extends Error {
```

Comment:
I was thinking we could literally just inline `EmptySuiteError` into `TestRunner.js` because we don't really need it anywhere else.